test/Microsoft.AspNet.Mvc.FunctionalTests/ModelBindingTest.cs

```diff
@@ -728,20 +728,21 @@ public class ModelBindingTest
 
             //Assert
             var json = JsonConvert.DeserializeObject<Dictionary<string, string>>(response);
+
             // 8 is the value of MaxModelValidationErrors for the application being tested.
             Assert.Equal(8, json.Count);
-            Assert.Equal("The Field1 field is required.", json["Field1.Field1"]);
-            Assert.Equal("The Field2 field is required.", json["Field1.Field2"]);
-            Assert.Equal("The Field3 field is required.", json["Field1.Field3"]);
-            Assert.Equal("The Field1 field is required.", json["Field2.Field1"]);
-            Assert.Equal("The Field2 field is required.", json["Field2.Field2"]);
-            Assert.Equal("The Field3 field is required.", json["Field2.Field3"]);
-            Assert.Equal("The Field1 field is required.", json["Field3.Field1"]);
+            Assert.Equal("The Field1 field is required.", json["Field1"]);
+            Assert.Equal("The Field2 field is required.", json["Field2"]);
+            Assert.Equal("The Field3 field is required.", json["Field3"]);
+            Assert.Equal("The Field4 field is required.", json["Field4"]);
+            Assert.Equal("The Field5 field is required.", json["Field5"]);
+            Assert.Equal("The Field6 field is required.", json["Field6"]);
+            Assert.Equal("The Field7 field is required.", json["Field7"]);
```

Comment:
I was wrong.  this test was demonstrating an unintended side-effect of `MutableObjectModelBinder` creating non-top-level POCO objects even when no data was available.  the correct thing is for required properties in the top-level object to be validated, as happens now.